index.js

```diff
@@ -0,0 +1,4 @@
+import Simctl from './lib/simctl';
```

Comment:
if you add it then make sure package.js is pointing to it, but I'd say lets keep everything as it was